utils/search.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
I would call this file `utils/records-decorator.js` or `service/records-decorator.js`